modules/api_docs/README.md

```diff
@@ -0,0 +1,36 @@
+# API Documentation
+
+## Purpose
+
+The API Documentation module leverages the Swagger UI structure to document the LORIS REST 
```

Comment:
This paragraph says it's just to describe the LORIS REST API, but then the interactions say any module can describe its API and gives an example of how to do it. As far as I know the intention is for the module to be used for the latter?

I think this paragraph needs to be written a little more broadly.

Reply:
@driusan done.